src/frontend/main/MapView.tsx

```diff
@@ -80,6 +72,7 @@ const Label = styled.label`
 const MapWrapper = styled.div`
   text-align: center;
   position: relative;
+  height: calc(100vh - 185px);
```

Comment:
We could add a comment here about where 185px comes from. I know, but it might not be obvious for someone who sees this for the first time 😅 Later we could even have them somewhere as constants, so people don't forget to change it if (or rather when) the layout changes.